here-naksha-app-service/src/main/resources/swagger/openapi.yaml

```diff
@@ -844,6 +844,8 @@ paths:
         - $ref: '#/components/parameters/TagList'
         - $ref: '#/components/parameters/Limit'
         - $ref: '#/components/parameters/PropertiesQuery'
+        - $ref: '#/components/parameters/PropertiesSelection'
+        - $ref: '#/components/parameters/Clip'
```

Comment:
Don't add clip yet in the specification of this API, as we are not able to support it at present